pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -147,21 +147,38 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		Expect(resp.Allowed).To(BeTrue())
 	})
 
-	DescribeTable("path validation should fail", func(path string) {
-		Expect(validatePath(k8sfield.NewPath("fake"), path)).To(HaveLen(1))
+	DescribeTable("container disk path validation should fail", func(containerDiskPath string, expectedCause string) {
+		vmi := newBaseVmi(libvmi.WithContainerDisk("testdisk", "testimage"))
+		vmi.Spec.Volumes[0].ContainerDisk.Path = containerDiskPath
+
+		ar, err := newAdmissionReviewForVMICreation(vmi)
+		Expect(err).ToNot(HaveOccurred())
+
+		resp := vmiCreateAdmitter.Admit(context.Background(), ar)
+		Expect(resp.Allowed).To(BeFalse())
+		Expect(resp.Result.Details.Causes).To(HaveLen(1))
+		Expect(resp.Result.Message).To(Equal(expectedCause))
 	},
-		Entry("if path is not absolute", "a/b/c"),
-		Entry("if path contains relative elements", "/a/b/c/../d"),
-		Entry("if path is root", "/"),
+		Entry(nil, "a/b/c", "spec.volumes[0].containerDisk must be an absolute path to a file without relative components"),
+		Entry(nil, "/a/b/c/../d", "spec.volumes[0].containerDisk must be an absolute path to a file without relative components"),
+		Entry(nil, "/", "spec.volumes[0].containerDisk must not point to root"),
```

Comment:
The first argument of Entry() should be the description.

Same bellow.

Reply:
I think the parameters suffice in this case (the description becomes a concatenation of them in the case of nil), but I'll change it anyway